.github/workflows/integration.yml

```diff
@@ -16,9 +16,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [netcoreapp3.1,net5.0]
+        version: [netcoreapp3.1,net5.0,net6.0]
     steps:
       - uses: actions/checkout@v2
+
+      - uses: actions/setup-dotnet@v1
```

Comment:
same comment as above, no need for 3.1.x or 5.0.x

Reply:
`*.tt` files? 🤣

https://docs.microsoft.com/visualstudio/modeling/code-generation-and-t4-text-templates